cmd/pomerium/main.go

```diff
@@ -40,6 +42,12 @@ func main() {
 	}
 
 	if err := runFn(ctx); err != nil && !errors.Is(err, context.Canceled) {
+		// if the error was due to a bad request, return 0 as the exit code
+		if strings.Contains(err.Error(), "bad request") {
```

Comment:
terminal errors are supposed to implement `IsTerminal()` interface we should be able to check that vs error text, please let me double-check if its propagated up to this point